specification/frontdoor/resource-manager/Microsoft.Network/preview/2018-08-01-preview/frontdoor.json

```diff
@@ -1694,7 +1700,14 @@
         },
         "type": {
           "type": "string",
-          "readOnly": true,
```

Comment:
Why are these changes being made to an existing version, @MikhailTryakhov are SDKs for `2018-08-01` already released?

Reply:
@zhuangquin please move your changes in ```2018-10-01``` api version

Reply:
@MikhailTryakhov, 2018-11-01 is actual now, isn't it?

Reply:
This is a backwards-compatible addition.